client/src/components/ManageAccount/ManageAccount.js

```diff
@@ -68,15 +68,12 @@ const ManageAccountView = () => {
           <LoadingSpinner />
         ) : (
           <>
-            {errors.data && (
-              <SectionMessage type="error">
-                Unable to get your profile data
-              </SectionMessage>
-            )}
-            {errors.fields && (
-              <SectionMessage type="error">
-                Unable to get form fields
-              </SectionMessage>
+            {(errors.data || errors.fields) && (
+              <div className="manage-account-error">
+                <SectionMessage type="error">
+                  An error occurred loading your profile information.
```

Comment:
@edmondsgarrett, Reminder: Craig remembers y'all had a chat, and because this error is thrown if any data in this section fails to load, then the message should be changed to:

```suggestion
                  An error occurred loading your account information.
```

Yes? No?